code-studio/src/js/blockly_editor.js

```diff
@@ -0,0 +1,163 @@
+import CodeMirror from "codemirror";
+import "codemirror/mode/markdown/markdown";
+import "codemirror/addon/edit/closetag";
+import "codemirror/addon/edit/matchtags";
+import "codemirror/addon/edit/matchbrackets";
+import "codemirror/addon/fold/xml-fold";
+import "codemirror/mode/xml/xml";
+import "codemirror/mode/javascript/javascript";
+import "@cdo/vendor/js/codemirror.inline-attach";
+import "marked";
```

Comment:
I thought we had decided to hold off on ES6 until first finishing the pipeline work.